omniscidb/IR/Node.h

```diff
@@ -591,7 +591,14 @@ class TranslatedJoin : public Node {
     CHECK(false);
     return nullptr;
   }
-  const std::string& getFieldName(size_t i) const override { CHECK(false); }
+  const std::string& getFieldName(size_t i) const override {
+    CHECK(false);
+    // Check(false) macro have abort in Logger d-tor,
```

Comment:
Let's just return an empty string, nobody will ever actually access it (b/c of the CHECK), and that is consistent with the rest of the code base.

Reply:
You can also throw an exception instead of return to resolve the warning.

Reply:
Ok, I guess we can leave the abort if it builds fine w/ gcc and clang.